src/astarte-client/models/Interface/index.ts

```diff
@@ -208,11 +208,13 @@ class AstarteInterface {
     }
     this.description = validatedObj.description;
     this.documentation = validatedObj.documentation;
-    this.mappings = validatedObj.mappings.map((mapping) =>
-      validatedObj.type === 'datastream'
+    this.mappings = validatedObj.mappings.map((mapping) => {
+      return validatedObj.type === 'datastream'
```

Comment:
if the function has only a single return statement, it can keep using the `() => ()` syntax